lib/Jsrt/Jsrt.cpp

```diff
@@ -3452,6 +3452,15 @@ CHAKRA_API JsSetPromiseContinuationCallback(_In_opt_ JsPromiseContinuationCallba
     /*allowInObjectBeforeCollectCallback*/true);
 }
 
+CHAKRA_API JsSetHostPromiseRejectionTracker(_In_ JsHostPromiseRejectionTrackerCallback promiseRejectionTrackerCallback, _In_opt_ void *callbackState)
```

Comment:
This needs to be moved into the `_CHAKRACOREBUILD` section at the end of the file.